Challenges/challenge_10_making_a_loading_bar/greghilston/src/script.js

```diff
@@ -0,0 +1,60 @@
+const oneSecondMiliSeconds = 1000
+let progressBar = document.getElementById("progress") 
```

Comment:
nit:
```suggestion
const progressBar = document.getElementById("progress")
const resetLoadingBarButton = document.getElementById("setload")
```
It's a good practice to always use `const` unless they will be re-assigned.